client/src/legacy/LeftAndMain.FieldHelp.js

```diff
@@ -10,28 +10,40 @@ $.entwine('ss', function($) {
    * input fields (e.g. GridField), and aren't compatible
    * with showing tooltips.
    */
-  $(".cms .field.cms-description-tooltip").entwine({
+  $(".cms .field.cms-description-tooltip .form__field-holder").entwine({
     onmatch: function() {
       this._super();
 
-      var descriptionEl = this.find('.description'), inputEl, tooltipEl;
+      var descriptionEl = this.find('.form__field-description');
       if(descriptionEl.length) {
+        var offset = this.css('padding-left');
         this
           // TODO Remove title setting, shouldn't be necessary
           .attr('title', descriptionEl.text())
-          .tooltip({content: descriptionEl.html()});
+          .tooltip({
+            content: descriptionEl.html(),
+            position: {
+              my: "left+" + offset + " top"
+            }
+          });
         descriptionEl.remove();
       }
     },
   });
 
   $(".cms .field.cms-description-tooltip :input").entwine({
     onfocusin: function(e) {
-      this.closest('.field').tooltip('open');
+      var holder = this.closest('.form__field-holder');
+      if (holder.data('tooltip')) {
```

Comment:
Prevents “calling open prior to initialization” errors

Reply:
I thought it was this way to check if it has been initialised, since it's jQuery UI and not jQuery core
```
if($element.data('ui-tooltip')) {
    $element.tooltip('destroy');
}
```

Reply:
oh, maybe it's because of how old the jQuery UI version is :/